oct_converter/readers/fda.py

```diff
@@ -155,6 +166,52 @@ def read_fundus_image_gray_scale(self):
         fundus_gray_scale_image = FundusImageWithMetaData(image)
         return fundus_gray_scale_image
 
+    def read_segmentation(self):
+        """
+        Reads layer segmentation data.
+
+        Segmentation values are returned in a dictionary with a key for every
+        layer boundary and are measured in pixels from B-scan bottom.
+
+        Returns
+        -------
+            seg_dict: dictionary with segmentation data.
+        """
```

Comment:
Please could you change to the following to make compatible with the other docstrings:

```
        Returns
            dict: dictionary with segmentation data.
```